### PR TITLE
IGNITE-17719 IgnitePdsThreadInterruptionTest#testInterruptsOnWALWrite hangs

### PR DESCRIPTION
**Pull request by tkalkirill**
https://issues.apache.org/jira/browse/IGNITE-17719

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java [hunk 1]
```diff
@@ -83,105 +80,97 @@ public class IgnitePdsThreadInterruptionTest extends GridCommonAbstractTest {
...
 
         for (Thread worker : workers)
             worker.start();
 
-        // Interrupts should not affect reads.
-        for (int i = 0; i < workers.length / 2; i++)
-            workers[i].interrupt();
-
-        U.sleep(3_000);
+        startThreadsLatch.await();
```
Comment (sashapolo) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java:161]: I think this should be a timed wait

Comment (tkalkirill) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java:161]: Fix it

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java [hunk 2]
```diff
@@ -158,7 +158,7 @@ public void testInterruptsOnRead() throws Exception {
         for (Thread worker : workers)
             worker.start();
 
-        startThreadsLatch.await();
+        startThreadsLatch.await(3, TimeUnit.SECONDS);
```
Comment (sashapolo) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java:161]: you should also add `assertTrue` here)

Comment (tkalkirill) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/file/IgnitePdsThreadInterruptionTest.java]: Fix it

**Pull request merged**